cmake/godotcpp.cmake

```diff
@@ -147,7 +146,26 @@ function( godotcpp_options )
 
     # Enable Testing
     option( GODOT_ENABLE_TESTING "Enable the godot-cpp.test.<target> integration testing targets" OFF )
-
+    # Define which targets create for build. By default all targets are included
+    # Not presented in SCons
+    option(GODOT_ADD_TARGET_TEMPLATE_RELEASE "Add template_release target to build" ON)
+    option(GODOT_ADD_TARGET_TEMPLATE_DEBUG "Add template_debug target to build" ON)
+    option(GODOT_ADD_TARGET_EDITOR "Add editor target to build" ON)
+    if(GODOT_ADD_TARGET_TEMPLATE_DEBUG)
+        list(APPEND GODOT_TARGETS "template_debug")
+    endif()
+    if(GODOT_ADD_TARGET_TEMPLATE_RELEASE)
+        list(APPEND GODOT_TARGETS "template_release")
+    endif()
+    if(GODOT_ADD_TARGET_EDITOR)
+        list(APPEND GODOT_TARGETS "editor")
+    endif()
+    if(NOT GODOT_TARGETS)
+        message(FATAL_ERROR "No targets were chosen to be build.See GODOT_ADD_TARGET_* variables: at least one of the should be ON")
```

Comment:
Slight grammar comment: `to be built` with a `t`. And a space after the full stop would be cool too.